doc/dev/release.md

```diff
@@ -216,6 +238,8 @@ Call the script with the only argument being the new SDK version:
 $ ./release.sh v1.3.0
 ```
 
+**NOTE** The `./release.sh` script requires `GNU sed` and the `GNU make` instead of the default installed sed. Install them with: `brew install gnu-sed` and `brew install make` and then, ensure they are present in your `$PATH`.
```

Comment:
```suggestion
**NOTE** The `./release.sh` script requires `GNU sed` and the `GNU make` instead of the default installed sed. Install them with: `brew install gnu-sed` and `brew install make`, then ensure they are present in your `$PATH`.
```